scipy/linalg/special_matrices.py

```diff
@@ -469,6 +469,51 @@ def kron(a, b):
     return np.concatenate(np.concatenate(o, axis=1), axis=1)
 
 
+def Khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    The result is the block matrix.
+
+    Mathematically:
```

Comment:
This section can't be here; the math should go in a `Notes` section under the `Returns` one. Please have a look at other functions for examples to follow. The docstring standard is documented at https://numpydoc.readthedocs.io/en/latest/format.html

Making that change should make the build-docs CI job pass

Reply:
done